docs/source/concepts.rst

```diff
@@ -1,2 +1,78 @@
 Key Concepts
 ============
+
+Let's first recap our example reporting workflow:
```

Comment:
there is no such thing as redundancy in documentation, and we have no guarantee someone will read it in the order we suggested.  totally reasonable for someone to read the key concepts section first, and we should have an example there waiting for them.  Otherwise, aren't the bullet points just as redundant as the image?:P